script/default-site

```diff
@@ -7,11 +7,10 @@ echo "$0: setting up tmp directory"
 mkdir -p ./tmp
 rm -Rf ./tmp/default-site
 
-workdir=$(pwd)
-
 echo "$0: creating new default site"
-bundle exec ruby exe/jekyll new tmp/default-site
+bundle exec jekyll new tmp/default-site
 pushd tmp/default-site
+workdir="../../"
 
 echo "$0: respecifying the jekyll install location"
 ruby -e "contents = File.read('Gemfile'); File.write('Gemfile', contents.sub(/gem \"jekyll\".*\\n/, 'gem \"jekyll\", path: \"$workdir\"'))"
```

Comment:
Essentially, the same thing, but much better.
Removes the variable and shortens the line here by 2 characters :stuck_out_tongue_winking_eye: 

I'll update shortly.